compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -509,7 +509,6 @@
     - hint: {lhs: 0 /= rem n 2, rhs: odd n}
     - hint: {lhs: not (even x), rhs: odd x}
     - hint: {lhs: not (odd x), rhs: even x}
-    - hint: {lhs: x ** 0.5, rhs: sqrt x}
```

Comment:
Is it worth still removing the lint?

Reply:
good catch, left that in by accident 1b270b0ac28439cf09a9853852a13ff3a640534d